gunicorn/http/wsgi.py

```diff
@@ -139,6 +139,9 @@ def create(req, sock, client, server, cfg):
             host = hdr_value
         elif hdr_name == "SCRIPT_NAME":
             script_name = hdr_value
+        elif hdr_name in ("SCRIPT_URL", "REDIRECT_URL"):
```

Comment:
looking at the django path , maybe we should handle these values differently though. Thoughts?